pkgs/development/libraries/mesa/default.nix

```diff
@@ -185,6 +191,18 @@ self = stdenv.mkDerivation {
     # move libOSMesa to $osmesa, as it's relatively big
     mkdir -p $osmesa/lib
     mv -t $osmesa/lib/ $out/lib/libOSMesa*
+  '' + lib.optionalString enableOpenCL ''
+    if [ -n "$(shopt -s nullglob; echo "$out"/lib/libMesaOpenCL*)" ]; then
```

Comment:
Is this still required with the `enableOpenCL` approach?
The moved files are also unconditionally referenced below.

Reply:
No, I just copied it from the version that didn’t use the enable flag without realising that.